compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -917,7 +951,8 @@ static void getDefaultMatmulWorkgroupSizes(linalg::LinalgOp op,
                                            int64_t vectorSize) {
   auto targetAttr = IREE::HAL::ExecutableTargetAttr::lookup(op);
   if (isX86(targetAttr)) {
-    sizes.append({8, 32, 16});
+    // 16 accumulators, 32 broadcasts, 8 vector loads.
+    sizes.append({8, 32, 4});
```

Comment:
How does `16 accumulators` map to K=4 config?

Reply:
I'll improve the doc. The number of vector register accumulators is determined by how we vectorize/unroll `C[i][j] +=`. Tile sizes for 'i' and 'j' dims are 8 and 32. Assuming f32 and 512-bit vector registers, 'j' is vectorized so 32 elements result in 2 vector register accumulator times 'i' unroll factor of 8.